### DISCUSSION TITLE
ShyFox 4.0 js mod teaser

### DISCUSSION THREAD

**Discussion by Naezr** [Naezr/ShyFox#176] [Announcements]
...

**Comment by Naezr** [comment]
[Запись экрана от 2024-10-09 15-58-48.webm](https://github.com/user-attachments/assets/004fa7e9-723d-4ad6-957c-4094d5d474aa)

**Reply by Naezr** [reply]
Why didn't I switch to fx-autoconfig earlier? That was a mistake.

**Reply by Halbour** [reply]
This looks dope, the new link system and that is actually cool af

**Comment by Naezr** [comment]
[Запись экрана от 2024-10-09 19-33-29.webm](https://github.com/user-attachments/assets/9ae39ddf-9093-4356-97d9-0280b4285255)

Essentially the Firefox developers could allow you to move these buttons, but for some reason they don't do so

**Comment by Naezr** [comment]
[Запись экрана от 2024-10-09 23-22-34.webm](https://github.com/user-attachments/assets/f7a48e7f-bd3b-41e1-98ce-d11fd10416a6)

**Comment by Naezr** [comment]
[Запись экрана от 2024-10-14 15-06-28.webm](https://github.com/user-attachments/assets/bd64f328-6de7-4526-9d73-a8e9c8d8478a)
